### PR TITLE
[CDAP-13583] Allow users to set up a pipeline for scoring data using a model

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-13583
Build: https://builds.cask.co/browse/CDAP-UDUT10-1

### CODE REVIEW

File: cdap-ui/app/cdap/components/Experiments/DetailedView/AddModelToPipelineBtn/index.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+          }
+        },
+        () => {
+          this.setState({
+            error: ERROR_MSG
+          });
+        }
+      );
+  }
+  geneartePipelineConfig = () => {
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Experiments/DetailedView/AddModelToPipelineBtn/index.js]: function typo

**Review by elfenheart — APPROVED** [review]
👍  LGTM

**Comment by ajainarayanan** [comment]
We decided to add the source and wrangler plugins as well while creating a scoring pipeline. Working on this.

**Review by bdmogal — APPROVED** [review]

File: cdap-ui/app/cdap/components/Experiments/DetailedView/AddModelToPipelineBtn/index.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  });
+
+  componentDidMount() {
+    this.setWorkspaceId();
+    this.fetchArtifactForPipelines();
+  }
+
+  setWorkspaceId = () => {
+    let workspaceId;
+    let {directives, srcPath} = this.props;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Experiments/DetailedView/AddModelToPipelineBtn/index.js]: These variables don't seem to be required to be defined here. Can be inside the callback function instead?

File: cdap-ui/app/cdap/components/Experiments/DetailedView/AddModelToPipelineBtn/index.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+            if (artifact.name === GLOBALS.etlDataPipeline) {
+              datapipelineArtifact = artifact;
+            }
+            if (artifact.name === GLOBALS.wrangler.pluginArtifactName) {
+              wranglerArtifact = artifact;
+            }
+            // FIXME: We need to move this to use some constant.
+            if (artifact.name === 'core-plugins') {
+              corepluginsArtifact = artifact;
+            }
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Experiments/DetailedView/AddModelToPipelineBtn/index.js]: would `switch` statement be better?

**Review by elfenheart — APPROVED** [review]
👍  LGTM

**Pull request merged**